react-project/src/reducers/treemapSlice.js

```diff
@@ -187,15 +231,20 @@ const treemapSlice = createSlice({
         action.payload.path !== state.mainTreemap.currentStatsPath
       ) {
         const newPath = `${action.payload.path}`;
-        const pathQuery = `$..[?(@.path=='${newPath}')]`;
-        let newData = getDataWithPathQuery(fullData, pathQuery, []);
-        console.log("scopeStatsIn", newData, pathQuery);
+        // TODO: delete?
+        let newData = goThroughTree(state.tree, newPath);
+        console.log("scopeStatsIn", newData, newPath);
```

Comment:
just here for testing purposes I guess, but needs to be removed once we go into production